### DISCUSSION TITLE
HuggingFace 🤗 Transformers is in Top 10 Python packages with the most unique contributors over the last 12 months on GitHub

### DISCUSSION THREAD

**Discussion by 8bitmp3** [huggingface/datasets#1692] [Show and tell]
HuggingFace 🤗 Transformers is in Top 10 Python packages with the most unique contributors over the last 12 months on GitHub

https://octoverse.github.com/static/github-octoverse-2020-community-report.pdf#page=18

![image](https://user-images.githubusercontent.com/19637339/103721538-ccea7080-4fc5-11eb-9eb3-e240e7657edd.png)

(Update: `huggingface/transformers` doesn't have Github Discussions, I think, so posting here @thomwolf @sguger @julien-c @patrickvonplaten et al.)

**Comment by julien-c** [comment]
Next year `datasets` will be in that Top 10!

**Comment by thomwolf** [comment]
Whououh!
